admin/src/main/scala/io/buoyant/admin/AdminConfig.scala

```diff
@@ -2,4 +2,4 @@ package io.buoyant.admin
 
 import io.buoyant.config.types.Port
 
-case class AdminConfig(port: Port = Port(9990))
+case class AdminConfig(port: Port = Port(9990), shutdownGraceMs: Int = 10000)
```

Comment:
Just a note to answer your question -- AdminConfig.port is probably wrong (throws NPEs) as it is here. It should probably be an `Option[Port]`.  Don't worry about fixing it here, I have follow branches that clean up admin initialization.